pkg/postage/listener/listener_test.go

```diff
@@ -515,19 +531,19 @@ func TestListenerBatchState(t *testing.T) {
 	}
 }
 
-func newEventUpdaterMock() (*updater, chan interface{}) {
-	c := make(chan interface{})
+func newEventUpdaterMock() *updater {
+	c := make(chan interface{}, 1)
```

Comment:
I recommend inlining the `make` call into the returned struct. The same is below.